### PR TITLE
Adding docker_admin_users to the docker_group

### PR DESCRIPTION
**Pull request by atosatto**
This PR is an abstraction of the work done by @arkadius started in #12.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +38,20 @@ Listening address where the raft APIs will be exposed.
 
 Listening port where the raft APIs will be exposed.
 
+    docker_admin_users:
+      - "{{ ansible_ssh_user }}"
```
Comment (arkadius) [inline README.md]: ansible_ssh_user is deprecated in favor to ansible_user regards to this: https://github.com/mitchellh/packer/issues/3275

File: tasks/docker_group.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  register: addtogroup
+  become: true
+  with_items: "{{ docker_admin_users }}"
+
+# Little hack to make docker group visible from http://stackoverflow.com/a/37337848/1370301
+- name: Kill the ansible_ssh_user active SSH connections
+  shell: "sleep 1; pkill -u {{ ansible_ssh_user }} {{ ssh_daemon }}"
+  async: 3  # fail after 3 seconds of waiting
+  poll: 2   # wait 2 seconds befor polling the task status
+  when: ansible_ssh_user in docker_admin_users
```
Comment (arkadius) [inline tasks/docker_group.yml]: the same as above

**Comment by atosatto** [comment]
Thank @arkadius for your review.
As suggested, I've renamed `ansible_ssh_user` to `ansible_user`.

**Comment by arkadius** [comment]
Looks good for me :+1:

**Pull request merged**